block/manager.go

```diff
@@ -79,15 +79,11 @@ type Manager struct {
 
 	blockCache *BlockCache
 
-	// blockStoreMtx is used by blockStoreCond
-	blockStoreMtx *sync.Mutex
-	// blockStoreCond is used to notify sync goroutine (SyncLoop) that it needs to retrieve blocks from blockStore
-	blockStoreCond *sync.Cond
+	// blockStoreCh is used to notify sync goroutine (SyncLoop) that it needs to retrieve blocks from blockStore
+	blockStoreCh chan interface{}
```

Comment:
Any reason you used an `interface`? 

I think all other uses in the codebase are a `struct`, so for consistency I'd keep it that way, unless there is a good reason to change to this model instead.

Reply:
No reason. Changed it to `struct`